roles/idrac_gather_facts/tasks/get_enclosure_emm_info.yml

```diff
@@ -8,3 +8,10 @@
 - name: Set enclosure EMM facts
   ansible.builtin.set_fact:
     enclosure_emm: "{{ emm_result.json.Members | ansible.utils.remove_keys(target=['@odata.context', '@odata.id', '@odata.type', 'Description', 'Links']) }}"
+  ignore_errors: true
+  register: emm_result_register
+
+- name: Set the fail message
```

Comment:
Don't fail it, skip it. Check for all the places.

Reply:
Done.